recipes/qcfractal/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "qcfractal" %}
+{% set version = "0.5.0rc1" %}
+{% set sha256 = "0c08b7c3b7097f75be86d77092a5dfdf31aa3e6e1818e05cddc25002e648e02a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Set-setup.py-to-zip_safe-False-for-Windows-testing.patch
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
From the [`setup.py`](https://github.com/MolSSI/QCFractal/blob/master/setup.py) there should be [`entry_points`](https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html?highlight=selectors#python-entry-points) specified here.

Reply:
Good catch on the entry points. I missed those!